dist/Xgreeter

```diff
@@ -2,6 +2,7 @@
 #
 # LightDM wrapper to run around greeter X sessions.
 
+export XCURSOR_SIZE=16
```

Comment:
```suggestion
```

Yeah, I don't think this should be here as this would enforce every system to have this cursor size (on web-greeter).